pages/02_auth_and_sso.md

```diff
@@ -8,38 +8,77 @@ In most embedding scenarios, you will want to enable single sign-on so that the
+
+1. After you have the JWT, you need to pass this value to the Tableau viz web component `<tableauViz>`. Once configured, users can securely view embedded content in your application without going through login screens.
+
+### External Authorization Servers
+
+If you are using an IdP on Tableau Server to authenticate users, you can use an external authorization server (EAS). The EAS must be set up to provide a JSON web token (JWT) for each user. You use the JWT when you embed the Tableau view as a web component in your application. When the embedded content is loaded, the standard OAuth flow is used. After users sign in to the IdP, they are automatically signed in to Tableau Server. For information, see [Register EAS to Enable SSO for Embedded Content (Linux)](https://help.tableau.com/current/server-linux/en-us/connected_apps_eas.htm) or [Register EAS to Enable SSO for Embedded Content (Windows)](https://help.tableau.com/current/server/en-us/connected_apps_eas.htm).
+
+### Add the JWT to the Tableau viz component
+
+Whether you are configuring your embedded web application to use EAS for Tableau Server, or as a connected app on Tableau Online or Tableau Server, you need to explicitly pass the JWT that is generated by the EAS or by your web server to the `<tableauViz>` web component.  You do this using the `token` attribute.
+
+For example, if you programmatically build the JWT for each user and assign it to a variable `JWT`, you might use a template literal to reference the JWT on your HTML page.
+
+```html
+
+<tableau-viz id="tableauViz"
+  src='http://your-tableau-server/views/my-workbook/my-view'
+  token="${JWT}">
+</tableau-viz>
+
+
+```
+
 ## Trusted Authentication
 
-Trusted authentication is, unlike the above options, a piece of functionality specific to Tableau Server. It allows you to trust specific machines to authenticate users on their behalf. Because the authentication happens with simple HTTP requests, it is the most versatile of the single sign-on options and can be used to integrate with, essentially, all other authentication systems.
+Trusted authentication is a piece of functionality specific to Tableau Server. It allows you to trust specific machines to authenticate users on their behalf. Because the authentication happens with simple HTTP requests, it is a versatile single sign-on option and can be used to integrate with, essentially, the other authentication systems (Active Directory, Kerbos, SAML, OpenID).
 
-[The Trusted Authentication documentation](https://onlinehelp.tableau.com/current/server/en-us/trusted_auth.htm) is a good resource for getting up and running, but below is a summary of the three steps in the trusted authentication workflow:
+[The Trusted Authentication documentation](http://onlinehelp.tableau.com/current/server/en-us/trusted_auth.htm) is a good resource for getting up and running, but below is a summary of the three steps in the trusted authentication workflow:
```

Comment:
Why removing https?

Reply:
Not sure how that happened.  Thanks for noticing.  There were a bunch of other links that used http on this page. Looks like I must have picked up an earlier version of this file. I'll investigate.